diff --git a/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec b/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec
index 34c4562..34c4562 100644
--- a/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec
+++ b/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.homepage = package['repository']['url']
   s.author = package['author']
   s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'Firebase/Auth', '~> 11.0'
-  s.dependency 'Firebase/Core', '~> 11.0'
-  s.dependency 'Firebase/Database', '~> 11.0'
+  s.dependency 'Firebase/Auth', '~> 10.25'
+  s.dependency 'Firebase/Core', '~> 10.25'
+  s.dependency 'Firebase/Database', '~> 10.25'
   s.swift_version = '5.1'
 enddiff --git a/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec b/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec
index 34c4562..34c4562 100644
--- a/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec
+++ b/node_modules/callkit-voip-capacitor-plugin/CallkitVoipCapacitorPlugin.podspec
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'Firebase/Auth', '~> 11.0'
-  s.dependency 'Firebase/Core', '~> 11.0'
-  s.dependency 'Firebase/Database', '~> 11.0'
+  s.dependency 'Firebase/Auth', '~> 10.25'
+  s.dependency 'Firebase/Core', '~> 10.25'
+  s.dependency 'Firebase/Database', '~> 10.25'
   s.swift_version = '5.1'
 end
